pkg/crypto/edg.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package crypto
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/ethersphere/bee/pkg/keystore"
+)
+
+// EDGSecp256_k1 aggregates private key cryptography functions that employ secp256k1
+type EDGSecp256_k1 struct{}
```

Comment:
How about if we have types unexported `edgSecp256_K1`, but singleton values exported `EDGSecp256_K1`?